README.md

```diff
@@ -3,7 +3,7 @@
 [Eppo](https://www.geteppo.com/) is a modular flagging and experimentation analysis tool. Eppo's PHP SDK is built to make assignments in multi-user server side contexts, compatible with PHP 7.3 and above. Before proceeding you'll need an Eppo account.
 
 ## Features
-
+- Bandits
```

Comment:
Consider listing this last as most customers will be more interested in the other features